packages/core/src/components/tabs/_tabs.scss

```diff
@@ -6,28 +6,50 @@ $tabs-border-size: 1px;
 
 @include exports('ray-tabs') {
   .#{$ray-class-prefix}tabs {
-    border-bottom: $tabs-border-size solid $ray-color-blue-50;
-    display: flex;
+    &__list {
+      border-bottom: $tabs-border-size solid $ray-color-gray-10;
+      display: flex;
```

Comment:
we should probably wrap these elements, we don't want the ui to break if they start overflowing